manifests/init.pp

```diff
@@ -6,9 +6,17 @@
   $certificates = hiera_hash(sslcert::certificates, undef),
   $certpath     = $sslcert::params::certpath,
   $keypath      = $sslcert::params::keypath,
+  $auto_certificate_renewal = $sslcert::params::auto_certificate_renewal_default_enabled,
+  $chainenabled = $sslcert::params::chainenabled,
 
 ) inherits sslcert::params {
 
   include ::sslcert::config
 
+
+  file { '/etc/facter/facts.d/facts-certificate-autorenew.yaml':
+    ensure  => present,
+    path    => '/etc/facter/facts.d/facts-certificate-autorenew.yaml',
```

Comment:
We don't need path as the name of the resource is the same.

Reply:
ensure => $ensure ?

Reply:
ensure aanpassen naar $ensure

Reply:
facts-sslcert-renewal.yaml